Malay-Kumar-Parida/15.Kubernetes/test_all.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment #Specifying wheteher deployment or service
```

Comment:
kind: can be other things too apart from deployment/services. please correct this comment